content/en/docs/reference/node/kernel-version-requirements.md

```diff
@@ -0,0 +1,90 @@
+---
+content_type: "reference"
+title: Linux Kernel Version Requirements
+weight: 10
+---
+
+{{% thirdparty-content %}}
+
+Many features rely on specific kernel functionalities and have minimum kernel version requirements.
+However, relying solely on kernel version numbers may not be sufficient
+for certain operating system distributions,
+as maintainers for distributions such as RHEL, Ubuntu and SUSE often backport selected features
```

Comment:
@pacoxu we should list these vendors in alphabetical order. Willing to send in a follow up PR?